x-pack/plugins/observability_solution/logs_data_access/server/services/register_services.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger } from '@kbn/logging';
+import { createGetLogsRatesService } from './get_logs_rates_service';
+
+export interface RegisterServicesParams {
+  logger: Logger;
+  deps: {};
+}
+
+export function registerServices(params: RegisterServicesParams) {
+  return {
+    getLogsRatesService: createGetLogsRatesService(params),
```

Comment:
@cauemarcondes would this service be responsible for returning the timeseries as well? How to you plan to handle this?

Reply:
No, there should be another service something like `getLogsRatesTimeseriesService`